sdk/inc/azure/core/internal/az_config_internal.h

```diff
@@ -24,7 +24,7 @@ enum
  */
 enum
 {
-  _az_INT64_AS_STR_BUF_SIZE = 20,
+  _az_INT64_AS_STR_BUFFER_SIZE = 20,
```

Comment:
Please add comment as to how 20 was determined

Reply:
The comment about that is right above :)